controllers/homeRoutes.js

```diff
@@ -139,5 +154,10 @@ router.get('/login', (req, res) => {
   res.render('login');
 });
 
+router.get('/logout', (req, res) => {
+  req.session.logged_in = false
+  res.render('homepage',{ ...dataObject, logged_in: req.session.logged_in});
+});
+
```

Comment:
We actually already have a logout route in /controllers/api/userRoutes (which also makes sure a user is logged in before logging them out, although idk how that could be abused).

Reply:
Repurposed this to fetch the backend logout request.